packages/client/hmi-client/src/components/articles/selected-article-pane.vue

```diff
@@ -66,7 +66,9 @@ const addResourcesToProject = async (projectId: string) => {
 
 // TODO: May need more formatting than just replcing <p></p> in future
 const formatAbstract = (item: XDDArticle) =>
-	item.abstract.replace('<p>', '\n').replace('</p>', '\n') || '[no abstract]';
+	item.abstract !== undefined
+		? item.abstract.replace('<p>', '\n').replace('</p>', '\n')
+		: '[no abstract]';
```

Comment:
Wait, why not just render the HTML as is on the HMI?

Reply:
I wasnt sure how to do that so I just referenced what I saw other components doing. Let me look into that real quick!

Reply:
Don't worry about it.